x-pack/legacy/plugins/siem/server/lib/detection_engine/types.ts

```diff
@@ -96,11 +99,13 @@ export type RuleAlertParamsRest = Omit<
     last_success_message?: IRuleStatusAttributes['lastSuccessMessage'] | undefined;
   };
 
-export type OutputRuleAlertRest = RuleAlertParamsRest & {
+export type OutputRuleAlertRest = Omit<RuleAlertParamsRest, 'throttle'> & {
   id: string;
   created_by: string | undefined | null;
   updated_by: string | undefined | null;
   immutable: boolean;
+  tags: string[];
+  throttle?: string | null;
```

Comment:
I was curious why this change was required and I think I have an idea (at least for throttle, not sure why tags was added).

Without this change to the `OutputRuleAlertType`, the type check fails inside of [utils.test.ts](https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/rules/utils.test.ts#L37) if the throttle property is present because the test case is for a `fullRule` that includes a `throttle`, however the [expected output](https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/rules/utils.test.ts#L39-L97) for these test cases do not have throttle present.

Because `fullRule` now has a throttle property, the expected output would be that it also has a throttle property. The test cases for `transformRuleToAlert` inside of utils.test.ts should be updated to include `throttle`, and will allow the `OutputRuleAlertRest` type to remain unchanged.